deploy.sh

```diff
@@ -118,18 +118,78 @@ for proj in $projects; do # bash only
   echo "||| $0: $proj"
   echo "|||"
 
+  # Function to fetch secrets from Google Cloud Secret Manager (uses same name for env var and secret)
+  fetch_secrets_to_env() {
+    # Check if gcloud is available
+    if ! command -v gcloud &> /dev/null; then
+      echo "[ERROR] gcloud command not found, skipping secret fetching"
+      return 1
+    fi
+
+    echo "Fetching secrets from Google Cloud Secret Manager for environment: $env..."
+
+    # Define the list of secrets to fetch - names will be identical in env and Secret Manager
+    local secrets=(
+      "STATSIG_CLIENT_KEY"
+      "STATSIG_SERVER_KEY"
+    )
+
+    # Add environment-specific Sentry token
+    local sentry_token_name
+    case "$env" in
+      "prod")
+        sentry_token_name="SENTRY_AUTH_TOKEN_PROD"
```

Comment:
[nit] let's rename to `SENTRY_AUTH_TOKEN_PROD_DEMO` and `SENTRY_AUTH_TOKEN_STAGING_TEAM_SE` for clarity